src/Symfony/Component/Security/Core/Exception/SessionExpiredException.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Core\Exception;
+
+/**
+ * SessionExpiredException is thrown when session has been idle for a long time.
+ *
+ * @author Antonio J. García Lagar <aj@garcialagar.es>
+ */
+class SessionExpiredException extends AuthenticationException
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getMessageKey()
+    {
+        return 'Session has expired.';
```

Comment:
this key needs to be added in the translation files of the Security component (at least the English file which is the reference, and then any other language for which you are able to translate it yourself)